.coveragerc

```diff
@@ -0,0 +1,28 @@
+[run]
```

Comment:
I think this changed was due to something I did in the plots regression using some built-in Travis functions. Since we are going to use an easier method I will revert this file back to the develop branch